### PR TITLE
EOL Automate 1 docs

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

### Description

[Please describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [x] Spell Check
- [x] Local build
- [x] Examine the local build
- [x] All tests pass

### CODE REVIEW

File: chef_master/source/_templates/nav-docs.html [hunk 1]
```diff
@@ -2211,84 +2177,21 @@
...
-                "hasSubItems": false,
-                "url": "/automate_compliance_credentials.html"
-              },
-              {
-                "title": "Perform Compliance Scan",
-                "hasSubItems": false,
-                "url": "/perform_compliance_scan.html"
-              },
               {
                 "title": "Audit Cookbook",
```
Comment (tas50) [inline chef_master/source/_templates/nav-docs.html:2181]: Is this something we have documented with A2? It's not part of workflow.

File: chef_master/source/_templates/nav-docs.html [hunk 2]
```diff
@@ -2354,23 +2251,12 @@
...
                 "hasSubItems": false,
                 "url": "/delivery_server_backup.html"
               },
-              {
-                "title": "Capacity Planning",
-                "hasSubItems": false,
-                "url": "/scaling_capacity_planning_chef_automate.html"
-              },
               {
                 "title": "Disaster Recovery",
```
Comment (tas50) [inline chef_master/source/_templates/nav-docs.html]: With workflow in A2 the backups / disaster docs are the ones in the A2 docs

File: chef_master/source/_templates/nav-docs.html (hunk 1 above)
Comment (kagarmoe) [reply chef_master/source/_templates/nav-docs.html:2181]: Yes, but the field teams are used to finding the Audit Cookbook doc here, so I thought that I would keep it.

File: chef_master/source/_templates/nav-docs.html [hunk 3]
```diff
@@ -2211,84 +2177,21 @@
                 "hasSubItems": false,
                 "url": "/fips.html"
```
Comment (tas50) [inline chef_master/source/_templates/nav-docs.html]: FIPS should really get a new home now that there's no automate stuff in it. It's an infra doc now.

**Review by tas50 — APPROVED** [review]

**Comment by kagarmoe** [comment]
image

**Pull request merged**